File: app/src/main/java/com/nav/richkart/login/ForgotPasswordResponse.java
package com.nav.richkart.login;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ForgotPasswordResponse {


    @SerializedName("data")
    @Expose
    private String mForgotPasswordData;

    @SerializedName("message")
    @Expose
    private String message;

    @SerializedName("status")
    @Expose
    private String mStatus;


    public String getmForgotPasswordData() {
        return mForgotPasswordData;
    }

    public void setmForgotPasswordData(String mForgotPasswordData) {
        this.mForgotPasswordData = mForgotPasswordData;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getmStatus() {
        return mStatus;
    }

    public void setmStatus(String mStatus) {
        this.mStatus = mStatus;
    }
}

File: app/src/main/java/com/richkart/android/login/ForgotPasswordResponse.kt
package com.richkart.android.login

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ForgotPasswordResponse(

        @SerializedName("data")
        @Expose
        var mForgotPasswordData: String = "",

        @SerializedName("message")
        @Expose
        var message: String = "",

        @SerializedName("status")
        @Expose
        var mStatus: String = ""

)